coverage.py

```diff
@@ -113,7 +113,7 @@ def run_once(self, offset):
         batch = self.items_that_need_coverage.limit(
             self.workset_size).offset(offset)
 
-        if not batch:
+        if not batch.count():
```

Comment:
This is a Query object, not a list. I noticed that CoverageProvider scripts were running forever even after they were done.